mpas_analysis/ocean/ocean_modelvsobs.py

```diff
@@ -163,50 +183,59 @@ def ocn_modelvsobs(config, field):
         monthsvalue = constants.monthdictionary[timestring]
 
         if isinstance(monthsvalue, (int, long)):
-            modeldata = monthly_clim.sel(month=monthsvalue).mpasData.values
-            obsdata = dsData.sel(month = monthsvalue).observationData.values
+            modeldata = monthly_clim.sel(month=monthsvalue)[field].values
+            obsdata = dsData.sel(month=monthsvalue)[obsFieldName].values
         else:
-            modeldata = np.sum(constants.dinmonth[monthsvalue-1]*monthly_clim.sel(month=monthsvalue).
-        		               mpasData.values.T, axis=1) / np.sum(constants.dinmonth[monthsvalue-1])
-            obsdata = np.nansum(daysarray[monthsvalue-1, :, :]*dsData.sel(month = monthsvalue).
-                                observationData.values, axis=0) / np.nansum(daysarray[monthsvalue-1, :, :],
-                                axis=0)
+
+            modeldata = np.sum(
+                constants.dinmonth[monthsvalue-1] *
+                monthly_clim.sel(month=monthsvalue)[field].values.T,
+                axis=1) / np.sum(constants.dinmonth[monthsvalue-1])
```

Comment:
@xylar, I'm wondering if we should loosen the PEP8 requirement then because it is meant to serve us and make our code better for the long term.  If it fails to do that, it really isn't meeting its purpose.  My personal concern is clear code that is readable and I personally view PEP8 as just a vehicle and set of tools to help in that goal.  The *true* goal, of course, isn't pure PEP8 compliance for its own sake.  I'm sorry if I've oversold PEP8 in the past.  Please use your judgement, of course!